specs/signature-specification.md

```diff
@@ -64,13 +64,13 @@ Besides the [image manifest property requirements][image-manifest-property-descr
 
 The client should be able to discover all the signatures belonging to an artifact (such as image manifest) by using [OCI Distribution Referrers API][oci-distribution-referrers].
 OCI Distribution Referrers API returns a paginated list of all artifacts belonging to a target artifact (such as container images, SBoMs).
-The implementation can filter the Notary Project signatures by either using OCI Distribution Referrers API or using custom logic on the client.
-Each the Notary Project signature refers to a signature envelope blob.
+The implementation can filter the Notary signatures by either using OCI Distribution Referrers API or using custom logic on the client.
+Each the Notary signature refers to a signature envelope blob.
 
 ### Signature Filtering
 
 An OCI artifact can have multiple signatures, The Notary Project tooling uses annotations of the signature artifact to filter relevant signatures based on the applicable trust policy.
-The Notary Project signature manifest's `io.cncf.notary.x509chain.thumbprint#S256` annotation key MUST contain the list of SHA-256 fingerprints of certificate and certificate chain used for signing.
+The Notary signature manifest's `io.cncf.notary.x509chain.thumbprint#S256` annotation key MUST contain the list of SHA-256 fingerprints of certificate and certificate chain used for signing.
```

Comment:
It seems we need to revert those changes as per the latest discussion about naming and branding